### DISCUSSION TITLE
MTT S80支持么

### DISCUSSION THREAD

**Discussion by LotusRootMonkey** [zai-org/ChatGLM3#529] [General]
我买了国产显卡MTT S80，我是一个小白，请教一下，MTT S80好像是说CUDA on MUSA的，如何使用MTT S80运行本项目呢

**Comment by zRzRzRzRzRzRzR** [comment]
官方脚本不支持这个卡，没有做这个卡的适配

**Comment by ferrymo** [comment]
老哥 猛啊，不过SD应该是可以去玩玩的
